packages/theme-check-docs-updater/src/themeLiquidDocsManager.ts

```diff
@@ -56,6 +60,13 @@ export class ThemeLiquidDocsManager implements ThemeDocset, JsonSchemaValidators
     );
   });
 
+  sectionSchema = memo(async (): Promise<string> => {
+    return this.loadFile(
+      'section_schema',
+      await readFallbackFile('section_schema.json').catch((_) => ''),
+    );
```

Comment:
That stuff is kind of messy because of the fallback to string and parse crap. Should clean up before merging.